CESC-submission/paper.tex

```diff
@@ -280,11 +277,11 @@ \section{Impact of Financial Incentives on Service Delivery}
 
 
-Another issue is regarding the payment computation. During early stages when the network launches, we anticipate a limited adoption and service demand. Thus, we reward Ursulas for just being online to encourage adoption. While in later stages, payments will be dispensed in proportion to the actual amount of service an Ursula provides. This is due to the fact that later on the inflation rewards will disappear, i.e., all tokens in the network will be minted, and the service fees are the only source of income. To incentivize Ursula to provide a correct and timely service, these rewards should be computed based on the actual amount of service. This shows how changes in incentive alignment lead to different views on how to distribute these incentives and ensure the security of the system.
+Another issue is regarding the payment computation. During early stages when the network launches, we anticipate a limited adoption and service demand. Thus, we reward Ursulas for just being online to encourage adoption. While in later stages, payments will be dispensed partially in proportion to the actual number of service requests an Ursula answers. This is due to the fact that later on the subsidies will effectively disappear, i.e., the majority of tokens in the network will be minted, and the service fees are the only source of income. To incentivize Ursula to provide a correct and timely service, these fees should be computed based on the actual amount of service. This shows how changes in incentive alignment lead to different views on how to distribute these incentives and ensure the security of the system.
 
 
 \section{Conclusion}
-While centrally-managed systems provide reliable services, they introduce trust, cost, and transparency issues. This motivated developing non-traditional peer-assisted models to create distributed resource markets that are open to anyone to join and serve others while collecting payments in return. However, such a paradigm comes with security and performance challenges that need to be addressed before having any practical deployment. Our work puts forward a generic framework for designing efficient and secure resource markets. It also examines in a greater depth how financial incentives affect security measures deployment and system design choices.
+While centrally-managed systems provide reliable services, they introduce trust, cost, and transparency issues. This motivated developing non-traditional peer-assisted models to create distributed resource markets that are open to anyone to join and serve others while collecting payments in return. However, such a paradigm comes with security and performance challenges that need to be addressed before having any practical deployment. Our work puts forward a generic framework for designing efficient and secure resource markets. It also examines in greater depth how financial incentives affect security measures and system design choices.
```

Comment:
affect security measures deployment and system -> affect security measures and system